interface/forms/CAMOS/help.html

```diff
@@ -163,7 +163,7 @@ <h1>Advanced Use</h1>
 /*billing::CPT4::80054::CMP:: :: ::50.00*/ /*add a cmp for $50*/
 /*billing::CPT4::85024::CBC:: :: ::25.00*/ /*add a cbc*/
 /*billing::CPT4::80061::Lipid Panel:: :: ::50.00*/
-/*billing::ICD9::V70.0::physical:: :: ::0.00*/ /*this is a diagnosis, not a procedure*/
+/*billing::ICD10::V70.0::physical:: :: ::0.00*/ /*this is a diagnosis, not a procedure*/
```

Comment:
use `Z00.00` instead?

https://www.icd10data.com/Convert/V70.0

Reply:
thank you @sjpadgett !

Reply:
cool thanks

Reply:
if I knew more about how this form works i'd restyle it. At least that's the excuse i'm going with like, ya know man!
Golden rules to follow: You never say you own a truck, know how to work on cars and most importantly, computers umm, what's that!

Reply:
https://user-images.githubusercontent.com/5618372/107997770-9be06100-6fb1-11eb-9a31-f624e448dfed.mp4

Reply:
lol, I know some folks i'd like to drag and drop like that.